README.md

```diff
@@ -25,3 +25,58 @@
 
 ## Tutorials and info
 * [Desktop Goose Modding Tutorial](https://www.youtube.com/watch?v=5OQiVWJ-TGc) by iwanMods (external link)
+### Hatgoos Tutorial
+Author of this mod is DaNike#6223
+
+This mod allows you to have your goose wear hats it can also be run with other mods
+
+<br/>
```

Comment:
Not needed here + avoid using HTML

Reply:
I removed the <br/> parts and change the image link to the correct format just now. Thank you for showing me the correct guide.